waagent

```diff
@@ -5909,8 +5916,14 @@ def main():
         else:
             args.append(a)
     global Config
-    Config = ConfigurationProvider()
+    Config = ConfigurationProvider(conf_file)
     
+    logfile = Config.get("Logs.File")
+    if logfile is not None:
+        myLogger.file_path = logfile
+    logconsole = Config.get("Logs.Console")
+    if logconsole is not None and logconsole.lower().startswith("y"):
```

Comment:
That's exactly what you're doing: the installation procedure replaces the old config with the new one, moving the existing config to `/etc/waagent.conf.old`
From README:
```
-install: Manual installation of the agent
     * Checks the system for required dependencies
     * Creates the SysV init script (/etc/init.d/waagent), the logrotate
       configuration file (/etc/logrotate.d/waagent) and configures the image
       to run the init script on boot
     * Writes sample configuration file to /etc/waagent.conf
     * Any existing configuration file is moved to /etc/waagent.conf.old
     * Detects kernel version and applies the VNUMA workaround if necessary
     * Moves udev rules that may interfere with networking
       (/lib/udev/rules.d/75-persistent-net-generator.rules,
        /etc/udev/rules.d/70-persistent-net.rules) to /var/lib/waagent/
```